docs/examples/basic-ssh-username-private-key-passphrase.yaml

```diff
@@ -0,0 +1,47 @@
+  # fA9YqxJjzoZy5870KOXY2kltlq/U/4Lrb0k75ag6ZVbi0oemACN6KCHtE+Zm2dac
+  # rW8oKWpRTbsvMOYUvSjF0u8BCrestpRUF977Ks8CgYEAicbLFCjK9+ozq+eJKPFO
+  # eZ6BU6YWR2je5Z5D6i3CyzT+3whXvECzd6yLpXfrDyEbPTB5jUacbB0lTmWFb3fb
+  # UK6n89bkCKO2Ab9/XKJxAkPzcgGmME+vLRx8w5v29STWAW78rj/H9ymPbqqTaJ82
+  # GQ5+jBI1Sw6GeNAW+8P2pLECgYAs/dXBimcosCMih4ZelZKN4WSO6KL0ldQp3UBO
+  # ZcSwgFjSeRD60XD2wyoywiUAtt2yEcPQMu/7saT63HbRYKHDaoJuLkCiyLBE4G8w
+  # c6C527tBvSYHVYpGAgk8mSWkQZTZdPDhlmV7vdEpOayF8X3uCDy9eQlvbzHe2cMQ
+  # jEOb9QKBgG3jSxGfqN/sD8W9BhpVrybCXh2RvhxOBJAFx58wSWTkRcYSwpdyvm7x
+  # wlMtcEdQgaSBeuBU3HPUdYE07bQNAlYO0p9MQnsLHzd2V9yiCX1Sq5iB6dQpHxyi
+  # sDZLY2Mym1nUJWfE47GAcxFZtrVh9ojKcmgiHo8qPTkWjFGY7xe/
+  # -----END RSA PRIVATE KEY-----
+  privateKey: "LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBbmdXTVluZGE5dkQydXR2YkFkZ0NPTFZOYW5BL01XNTBlcjVST1cyMWl0L2VwaDF1CjZSQ3VaMENpdVlVRTVFYjhrT09RUDdNVEwzSXh5djlHVzZobU1ad2p5dmNDYW1LajdjWXVFSEJZa24wWDJKZ3cKc3lQR1VXWndJVGdTeGdiL1ZmalJLYkF0VWR2WE5Gakh4a25VbGFWZCtHNmdRcE41THYzLy9PL0VnbG1WcWYxZApDTTJ4QXk5SXhrOXJvTVNtQnBnd0M3bENzaTFXOUlHZExyakxBQzk2QnJKa0hYMUVEUURkQjh0V2c4cUxqWmZyCkwxaW9kZEcvTkRIOGxPVWV0V1g5U0I1V0Y0eGkvb0JSTnZTQ3dtQkFhOHYyRHZoUy9URXdjV0FzUmVjbFJDTlcKNWVHQXFoYmIwS2w4RTBoWUpkRmxFS1lqUUgzeTVjWnRxTUFpdXdJREFRQUJBb0lCQUdRSzJUVGhvWXBqUmFGSgpYWjhPTldIWGpwcUxVOGFreWtPSFIvOFdzTytxQ2RpYkc4T2NGdjR4a3BQblhoQnp6S1NpSFlubWdvZndRUXZtCmo1R3B6SUV0L0E4Y1VNQXZrTjhSTDhxaWhjREFSNStOd284M1grL2E3YlJxUHFCMmY2TGJNdmkwbkF5T0pQSDAKSHc0dllkSVg3cVZBekY4NTVHZlcwUUUrZnVlU2R0Z1d2aUpNOGdaSGRoQ3FlL3pxWW0wMTZ6TmFhdmFwNTMwcgp0Si8rdmhVVzhXWXFKcUJXOCs1OGxhVzV2VEJ1c05zVmplTDQweUpGOFgvWFFRY2RaNFhtdGhOY2VneDc5b2ltCmo5RUx6WDB0dGNoaXdBZS90ckx4VGtkV2I0ckVGeitVNTBpQU9NVWRTOFQwYnJiNWJ4aHFOTS9CeWlxUTI4VzkKMk5KQ1ZFa0NnWUVBMHBoQ0U5aUtWV05abnZXWDYrZkhncjJOTzJTaFBleFBlUmZGeHIwdWdYR1RRdnlUMEhuTQovUS8vVitMZHVQTVg4YjJBc096STByUWgrNGJqb2hPWnZLbUdLaXVQdjNlU3ZxcGkvcjYyMDhaVlRCampGdkJPClVRaE1iUFV5UjZ2TzFyeUZEd0JNd01xUTA2bGRrWEFyaEIrU0cwZFluT0tiLzZnMG5PMkJWRlVDZ1lFQXdCZUgKSEdOR3V4d3VtNjNVQWFxeVg2bFJTcEdHbTZYU0NCaHp2SFVQblZwaGdxN25uWk9HbDB6M1U0OWpyZUN2dXVFYwpmQTlZcXhKanpvWnk1ODcwS09YWTJrbHRscS9VLzRMcmIwazc1YWc2WlZiaTBvZW1BQ042S0NIdEUrWm0yZGFjCnJXOG9LV3BSVGJzdk1PWVV2U2pGMHU4QkNyZXN0cFJVRjk3N0tzOENnWUVBaWNiTEZDaks5K296cStlSktQRk8KZVo2QlU2WVdSMmplNVo1RDZpM0N5elQrM3doWHZFQ3pkNnlMcFhmckR5RWJQVEI1alVhY2JCMGxUbVdGYjNmYgpVSzZuODlia0NLTzJBYjkvWEtKeEFrUHpjZ0dtTUUrdkxSeDh3NXYyOVNUV0FXNzhyai9IOXltUGJxcVRhSjgyCkdRNStqQkkxU3c2R2VOQVcrOFAycExFQ2dZQXMvZFhCaW1jb3NDTWloNFplbFpLTjRXU082S0wwbGRRcDNVQk8KWmNTd2dGalNlUkQ2MFhEMnd5b3l3aVVBdHQyeUVjUFFNdS83c2FUNjNIYlJZS0hEYW9KdUxrQ2l5TEJFNEc4dwpjNkM1Mjd0QnZTWUhWWXBHQWdrOG1TV2tRWlRaZFBEaGxtVjd2ZEVwT2F5RjhYM3VDRHk5ZVFsdmJ6SGUyY01RCmpFT2I5UUtCZ0czalN4R2ZxTi9zRDhXOUJocFZyeWJDWGgyUnZoeE9CSkFGeDU4d1NXVGtSY1lTd3BkeXZtN3gKd2xNdGNFZFFnYVNCZXVCVTNIUFVkWUUwN2JRTkFsWU8wcDlNUW5zTEh6ZDJWOXlpQ1gxU3E1aUI2ZFFwSHh5aQpzRFpMWTJNeW0xblVKV2ZFNDdHQWN4Rlp0clZoOW9qS2NtZ2lIbzhxUFRrV2pGR1k3eGUvCi0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCg=="
```

Comment:
Yuck. [displaying this stuff](https://itnext.io/secrets-in-plain-text-13a98f54ef97)

Reply:
I thought that it should be a working example (so already base64 encoded).
If this is displayed this as a plain text priv key, users will have to encode key before using it.

Reply:
Yes, I was just noting that you may want to link to the tool I mentioned as a way of helping people debug this mess.